controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/cost/restapi/CostJsonModelAdapter.java

```diff
@@ -0,0 +1,113 @@
+    }
+
+    public static CostJsonModel.Cluster toJsonModel(CostCluster clusterCost) {
+        CostJsonModel.Cluster cluster = new CostJsonModel.Cluster();
+        cluster.count = clusterCost.getClusterInfo().getHostnames().size();
+        cluster.resource = getResourceName(clusterCost.getResultUtilization());
+        cluster.utilization = clusterCost.getResultUtilization().getMaxUtilization();
+        cluster.tco = (int)clusterCost.getTco();
+        cluster.flavor = clusterCost.getClusterInfo().getFlavor().toString();
+        cluster.waste = (int)clusterCost.getWaste();
+        cluster.type = clusterCost.getClusterInfo().getClusterType().name();
+        cluster.util = new CostJsonModel.HardwareResources();
+        cluster.util.cpu = clusterCost.getResultUtilization().getCpu();
+        cluster.util.mem = clusterCost.getResultUtilization().getMemory();
+        cluster.util.disk = clusterCost.getResultUtilization().getDisk();
+        cluster.util.diskBusy = clusterCost.getResultUtilization().getDiskBusy();
+
+        Flavor flavor = clusterCost.getClusterInfo().getFlavor();
+        cluster.usage = new CostJsonModel.HardwareResources();
+        cluster.usage.cpu = flavor.getMinCpuCores() * clusterCost.getSystemUtilization().getCpu();
+        cluster.usage.mem = flavor.getMinMainMemoryAvailableGb() *  clusterCost.getSystemUtilization().getMemory();
+        cluster.usage.disk = flavor.getMinDiskAvailableGb() * clusterCost.getSystemUtilization().getDisk();
+        cluster.usage.diskBusy = clusterCost.getSystemUtilization().getDiskBusy();
+        cluster.hostnames = new ArrayList<>(clusterCost.getClusterInfo().getHostnames());
+
+        return cluster;
+    }
+
+    private static void toSlime(CostCluster clusterCost, Cursor object) {
+        object.setLong("count", clusterCost.getClusterInfo().getHostnames().size());
+        object.setString("resource", getResourceName(clusterCost.getResultUtilization()));
+        object.setDouble("utilization", clusterCost.getResultUtilization().getMaxUtilization());
+        object.setLong("tco", (int)clusterCost.getTco());
+        object.setString("flavor", clusterCost.getClusterInfo().getClusterType().name());
+        object.setLong("waste", (int)clusterCost.getWaste());
+        object.setString("type", clusterCost.getClusterInfo().getClusterType().name());
+        Cursor utilObject = object.setObject("util");
+        utilObject.setDouble("cpu", clusterCost.getResultUtilization().getCpu());
+        utilObject.setDouble("mem", clusterCost.getResultUtilization().getMemory());
+        utilObject.setDouble("disk", clusterCost.getResultUtilization().getDisk());
+        utilObject.setDouble("diskBusy", clusterCost.getResultUtilization().getDiskBusy());
+        Cursor usageObject = object.setObject("usage");
+        usageObject.setDouble("cpu", clusterCost.getSystemUtilization().getCpu() * 100);
```

Comment:
I guess we don't need * 100 here, do we? The metric from getSystemUtilization() is already computed with * 100.

Reply:
I went with ratios here to make the calculations easier, leaving percentage for the 'presentation'